.gitignore

```diff
@@ -1,5 +1,5 @@
 .env
```

Comment:
Should we also make the .giignore a template file so that people can add their own pathname? Or should we just add it to the documentation to go into this file and change it to their own pathname?

Reply:
hmmm I think it would be fine to just add it to the documentation since it's just like one more line